templates/candlepin.conf.erb

```diff
@@ -34,6 +34,9 @@ candlepin.crl.file=<%= scope['candlepin::crl_file'] %>
 candlepin.ca_key_password=<%= scope['candlepin::ca_key_password'] %>
 <%- end -%>
 
+# Default is every hour - override to once every 24h
+pinsetter.org.candlepin.pinsetter.tasks.ExpiredPoolsJob.schedule=0 0 0 * * ?
```

Comment:
Is this something that ought to be tunable per environment based on scale? Or is 24 hours generally good for all deployments?

Reply:
no need to tune, 24h is suitable for all environments.

Reply:
Is there some mechanism of spreading out? I'm worried about all clients checking in at the same time and overloading the server.

Reply:
Never mind, just realized this is the server.

I still do think @ehelms has a good point of making it a parameter. We don't need to expose it to the user, but it allows tuning via hiera. For example, if they need to reduce it even further or move it another hour than midnight.

Also looks like the tests need to be modified to expect this line.

Reply:
@ekohl updated as a param + tests!